packages/google_maps_flutter/google_maps_flutter_platform_interface/lib/src/method_channel/method_channel_google_maps_flutter.dart

```diff
@@ -520,7 +516,7 @@ class MethodChannelGoogleMapsFlutter extends GoogleMapsFlutterPlatform {
         },
       );
     }
-    return buildView(
+    return super.buildView(
```

Comment:
Why super? Super throws, unless I'm missing something.

I think it would be clearer to just move the current `buildView` implementation here, and then have `buildView` be a simple passthrough to `buildViewWithTextDirection` with a default paramater.

Reply:
Woops. This was suppose to be `super.buildViewWithTextDirection`. I changed it to your suggestion